src/Samples/AuthoringDemo/AuthoringDemo.sln

```diff
@@ -1,44 +1,74 @@
 	ProjectSection(ProjectDependencies) = postProject
 		{7E449DC2-5334-464D-B478-1715B96C8050} = {7E449DC2-5334-464D-B478-1715B96C8050}
 	EndProjectSection
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AuthoringDemo", "AuthoringDemo\AuthoringDemo.csproj", "{7E449DC2-5334-464D-B478-1715B96C8050}"
+EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{1452D9DE-635B-46F9-9C0E-B2FEB4F0A9EB}"
 	ProjectSection(SolutionItems) = preProject
 		Directory.Build.props = Directory.Build.props
 		Directory.Build.targets = Directory.Build.targets
 		nuget.config = nuget.config
 	EndProjectSection
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "WinUI3CppApp", "WinUI3CppApp\WinUI3CppApp.vcxproj", "{CA5A86BC-C0FB-472D-B1B9-3C5221595606}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|ARM64 = Debug|ARM64
 		Debug|x64 = Debug|x64
 		Debug|x86 = Debug|x86
+		Release|ARM64 = Release|ARM64
 		Release|x64 = Release|x64
 		Release|x86 = Release|x86
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{7E449DC2-5334-464D-B478-1715B96C8050}.Debug|x64.ActiveCfg = Debug|x64
-		{7E449DC2-5334-464D-B478-1715B96C8050}.Debug|x64.Build.0 = Debug|x64
-		{7E449DC2-5334-464D-B478-1715B96C8050}.Debug|x86.ActiveCfg = Debug|x64
-		{7E449DC2-5334-464D-B478-1715B96C8050}.Release|x64.ActiveCfg = Release|x64
-		{7E449DC2-5334-464D-B478-1715B96C8050}.Release|x64.Build.0 = Release|x64
-		{7E449DC2-5334-464D-B478-1715B96C8050}.Release|x86.ActiveCfg = Release|x64
+		{96752AD7-C344-4D4D-9E0B-39BED88B191A}.Debug|ARM64.ActiveCfg = Debug|ARM64
+		{96752AD7-C344-4D4D-9E0B-39BED88B191A}.Debug|ARM64.Build.0 = Debug|ARM64
 		{96752AD7-C344-4D4D-9E0B-39BED88B191A}.Debug|x64.ActiveCfg = Debug|x64
 		{96752AD7-C344-4D4D-9E0B-39BED88B191A}.Debug|x64.Build.0 = Debug|x64
 		{96752AD7-C344-4D4D-9E0B-39BED88B191A}.Debug|x86.ActiveCfg = Debug|Win32
 		{96752AD7-C344-4D4D-9E0B-39BED88B191A}.Debug|x86.Build.0 = Debug|Win32
+		{96752AD7-C344-4D4D-9E0B-39BED88B191A}.Release|ARM64.ActiveCfg = Release|ARM64
+		{96752AD7-C344-4D4D-9E0B-39BED88B191A}.Release|ARM64.Build.0 = Release|ARM64
 		{96752AD7-C344-4D4D-9E0B-39BED88B191A}.Release|x64.ActiveCfg = Release|x64
 		{96752AD7-C344-4D4D-9E0B-39BED88B191A}.Release|x64.Build.0 = Release|x64
 		{96752AD7-C344-4D4D-9E0B-39BED88B191A}.Release|x86.ActiveCfg = Release|Win32
 		{96752AD7-C344-4D4D-9E0B-39BED88B191A}.Release|x86.Build.0 = Release|Win32
+		{7E449DC2-5334-464D-B478-1715B96C8050}.Debug|ARM64.ActiveCfg = Debug|arm64
+		{7E449DC2-5334-464D-B478-1715B96C8050}.Debug|ARM64.Build.0 = Debug|arm64
+		{7E449DC2-5334-464D-B478-1715B96C8050}.Debug|x64.ActiveCfg = Debug|x64
+		{7E449DC2-5334-464D-B478-1715B96C8050}.Debug|x64.Build.0 = Debug|x64
+		{7E449DC2-5334-464D-B478-1715B96C8050}.Debug|x86.ActiveCfg = Debug|x64
```

Comment:
Mismatch in architecture. Make sure BuildConfiguration is set up right for the solution

Reply:
fixed